docs/index.rst

```diff
@@ -23,6 +23,25 @@ sunkit-image
           generated/gallery/index
           changelog
 
+Mission Statement
+=================
+
+
+The goal of ``sunkit-image`` is not to be a general purpose image processing library.
+
+The goal of ``sunkit-image`` is to provide access to image processing routines that are:
+
+1. Focused on being applied to solar image data.
+2. Are published in the literature or in preparation to be published.
+3. Widely used throughout the solar physics community.
+   Examples include co-alignment routines that compensate for incorrect pointing, solar feature identification algorithms, and filtering functions. 
+
+As this is a SunPy affiliated package, the inputs to these routines are required to be Maps (sunpy) or NDCubes (ndcube).
```

Comment:
Currently asdf breaks this.

THe time lag one as well, but maybe we should add time series too? :P

Reply:
What's the input to timelag? I thought it outputted time series?

Reply:
array-like inputs and outputs